state/user.ts

```diff
@@ -0,0 +1,20 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+type User = {
+  username: string;
+};
```

Comment:
uhm, what do you think about defining all the types in one types file?

Reply:
Yeah sure. It makes sense. Making the changes right away =)